bleak/backends/corebluetooth/client.py

```diff
@@ -354,14 +354,19 @@ async def start_notify(
         self,
         characteristic: BleakGATTCharacteristic,
         callback: NotifyCallback,
+        cb: Optional[Dict[str, Any]] = None,
```

Comment:
Would also be nice to make a `TypedDict` for this. (I don't mind doing that later though)